packages/component-library/stories/BarChart.story.js

```diff
@@ -1,48 +1,135 @@
 import React from "react";
 /* eslint-disable import/no-extraneous-dependencies */
 import { storiesOf } from "@storybook/react";
+import civicFormat from "../src/utils/civicFormat";
 
-import { withKnobs, text, object } from "@storybook/addon-knobs";
+import {
+  withKnobs,
+  text,
+  object,
+  boolean,
+  number,
+  optionsKnob as options
```

Comment:
I was curious about the naming of this knob -- It's unfortunate that Storybook has this knob with a different naming convention then all other knobs. Understandable, though.

Reply:
Yes, they are using a different naming convention.  Here is the documentation for it:
https://github.com/storybooks/storybook/tree/master/addons/knobs

Scroll down to the "options" knob and it has an example of using this knob.